test/send_reminders_test.js

```diff
@@ -20,16 +20,16 @@ var dates = require("../utils/dates"),
 nock.disableNetConnect();
 nock('https://api.twilio.com:443').log(console.log);
 
-describe("with a reminder that hasn't been sent", function() {
-    beforeEach(function(done) {
+describe("with one reminder that hasn't been sent", function() {
+    beforeEach(function (done) {
         manager.ensureTablesExist()
             .then(clearTable("cases"))
             .then(clearTable("reminders"))
-            .then(loadCases([turnerData()]))
-            .then(addTestReminder)
+            .then(loadCases([case1]))
+            .then(addTestReminders([reminder1]))
             .then(function(){
                 done();
-            });   
+            });
```

Comment:
you should put this code at the end so errors fail the test. right now, errors would just time out the test and wouldn't be reported (i think).
`.catch(done)`